hive/r/0-configure.R

```diff
@@ -0,0 +1,25 @@
+
+
+# packages ----------------------------------------------------------------
+# install.packages("RHive")
+library(RHive)
+
+
+# rhive -------------------------------------------------------------------
+dirHive <- "/user/hive"
+Sys.setenv(HIVE_HOME = "/usr/lib/hive")
+rhive.init()
+rhive.connect(host = "127.0.0.1", hiveServer2 = TRUE)
+
+
+# table parameters --------------------------------------------------------
+name_table <- "taxi_sample" # "/rmr2/trip_data"
+data_location <- "/user/andrie.devries/taxi/sample"
```

Comment:
And what happens if this runs not in my account?

Please make this general, independent of user.